src/pages/cart/filled-cart/filled-cart.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import Button from '@material-ui/core/Button';
+import { Link } from 'react-router-dom';
+import { useStyles } from './filled-cart.styles';
+import CartItem from '../cart-item';
+import {
+  CART_BUTTONS,
+  CART_TABLE_FIELDS
+} from '../../../translations/cart.translations';
+import { removeItemFromCart } from '../../../redux/cart/cart.actions';
+import { MODAL_DELETE_MESSAGES } from '../../../translations/modal.translations';
+import Modal from '../../../components/modal';
+
+const FilledCart = ({ items }) => {
+  const [modalVisibility, setModalVisibility] = useState(false);
+  const [modalItem, setModalItem] = useState({});
+  const dispatch = useDispatch();
+  const language = useSelector(({ Language }) => Language.language);
+  const styles = useStyles();
+
+  const totalCounter = () =>
+    items.reduce((acc, item) => acc + item.totalPrice * item.quantity, 0);
+
+  const onModalAction = (action) => {
+    action && dispatch(removeItemFromCart(modalItem));
+    setModalVisibility(false);
+  };
+
+  return (
+    <div className={styles.root}>
+      <div className={styles.table}>
+        <div className={styles.tableHeader}>
+          <div>{CART_TABLE_FIELDS[language].item}</div>
+          <div>{CART_TABLE_FIELDS[language].quantity}</div>
+          <div>{CART_TABLE_FIELDS[language].price}</div>
+        </div>
+        {items.map((item) => (
+          <CartItem
+            key={Math.random()}
+            item={item}
+            language={language}
+            setModalVisibility={setModalVisibility}
+            setModalItem={setModalItem}
+          />
+        ))}
+      </div>
+      <div className={styles.total}>
+        {CART_TABLE_FIELDS[language].total}: {totalCounter()} UAH
```

Comment:
Please, inline here body of the `totalCounter`, that code is used only once so no reason to create a function for it